src/platforms/apple/common/configuration/integrations/default.mdx

```diff
@@ -15,6 +15,12 @@ This integration is the core part of the SDK. It hooks into all signal and excep
 
 This integration will swizzle some methods to create breadcrumbs. For example, for `UIApplicationDidReceiveMemoryWarningNotification`, `sendAction:to:from:forEvent:` (UI interactions) or `viewDidAppear:` those breadcrumbs will be attached to your events.
 
+### SentryOutOfMemoryTrackingIntegration
+
+**Attention: This is a preview API available since 7.0.0-alpha.3. This API is not in stable state yet. It may be renamed, changed or even removed in a future version.**
```

Comment:
```suggestion
**Attention: This is a preview API available since 7.0.0-alpha.3. This API is not in stable state yet. It may be renamed, changed, or even removed in a future version.**
```